packages/playground/src/dashboard/twin_view.vue

```diff
@@ -264,13 +264,22 @@ function redirectToDao() {
 
 async function saveEmail() {
   try {
-    loading.value = true;
-    savingEmail.value = true;
-    profileManager.updateEmail(email.value);
-    await storeEmail(grid!, email.value);
-    editEmail.value = false;
-    loading.value = false;
-    savingEmail.value = false;
+    const balance: Balance = await loadBalance(grid!);
+    if (balance.free < 1) {
+      editEmail.value = false;
+      createCustomToast(
+        "Invalid Transaction: Inability to pay some fees, e.g. account balance too low",
+        ToastType.danger,
+      );
+    } else {
```

Comment:
instead of else, you can also `return nothing` in the above line 
```suggestion
   if (balance.free < 1) {
      editEmail.value = false;
      return;
    }
    editEmail.value = true;
```

but it is not actually comment you can ignore it :D